tests/integration/dump_test.go

```diff
@@ -603,3 +608,65 @@ func Test_Dump_KeysAndKeySets_Konnect(t *testing.T) {
 		})
 	}
 }
+
+// test scope:
+//
+// - >=3.10.0
+func Test_Dump_Deterministic_Sanitizer(t *testing.T) {
+	runWhen(t, "kong", ">=3.10.0")
```

Comment:
I will expand this in subsequent PRs.
Limited this for now as partials are being tested